### ISSUE TITLE
Improve enum support in GraphQL

### ISSUE THREAD

**Issue by shafreenAnfar** [ballerina-platform/ballerina-standard-library#1306]
Consider the below as something that can be converted to a GraphQL enum as well. Because Ballerina enum is simply a shorthand for the below. 

```ballerina
public const OPEN = "OPEN";
public const CLOSED = "CLOSED";
public const HOLD = "HOLD";

public type LiftStatus OPEN|CLOSED|HOLD;
```

**Comment by ThisaruGuruge** [comment]
We planned to implement this with #2047 but blocked on https://github.com/ballerina-platform/ballerina-lang/issues/33577

**Comment by ThisaruGuruge** [comment]
This is now related to #2047

**Comment by MohamedSabthar** [comment]
Blocked on [#38004](https://github.com/ballerina-platform/ballerina-lang/issues/38004)

**Comment by ThisaruGuruge** [comment]
Closing this as Deferred. Can be re-opened if this is needed.
